src/graphics/window.c

```diff
@@ -62,6 +62,19 @@ window_id window_get_id(void)
     return current_window->id;
 }
 
+void window_draw_old_behind(void)
```

Comment:
Naming: I think a better name would be `window_draw_underlying_window()`. Also, put this function at the very bottom so it's after `window_draw` which is related.

Reply:
I didn't quite like the name too. Nice name idea ;)